app/assets/stylesheets/sephora_style_guide/variables/_modal.scss

```diff
@@ -4,7 +4,7 @@ $modal-transition:            transform $modal-transition-time, opacity $modal-t
 $modal-content-bg:            $white;
 $modal-backdrop-bg:           $black;
 $modal-backdrop-opacity:      0.5 !default;
-$modal-backfrop-transition:   opacity $modal-transition-time;
+$modal-backdrop-transition:   opacity 0.15s;
```

Comment:
Why not still use the variable?

Reply:
If the variable (0.4s) is used, the transition is slower and it takes more time for the modal to show as compared to production. Previously the duration was set [within](https://github.com/luxola/sephora-style-guide/blob/2278352f2404d86f5eb83a75967b9aa39edf08a1/app/assets/javascripts/sephora_style_guide/modal.js#L29) `modal.js`, but v4.1.1 uses a [new utility function](https://github.com/luxola/sephora-style-guide/blob/a5c75067b7b8b57c389e541174cf87a19535a231/app/assets/javascripts/sephora_style_guide/util.js#L122) to get the transition duration from the actual element. 

I could make this a variable if it's better, style wise.